roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -23,6 +23,7 @@
   service: name=aci-gw state=started
   run_once: true
 
+# NOTE: Use Contivctl command when it is available
 - name: set aci mode
-  shell: contivctl global set -nwinfra aci
```

Comment:
just change it to `contivctl net` where `netctl` is used. that's how it's wrapped, or was.